docs/offline.md

```diff
@@ -0,0 +1,27 @@
+This document describes in more details how to use this library in offline mode.
+
+All applications do not require offline mode, and offline mode needs two large dependencies : `pouchdb` and `pouchdb-find`.
+As those two dependencies are expensive in size, we decided to not provide them by default, since v0.2.0.
+
+So if an app needs to use offline mode, it will need to add those dependencies by itself. It means that developers just need to do:
```

Comment:
When you say `yarn add pouchdb`, do you mean in the app directory or in the lib directory?